File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/BALANCA/ler_peso.dart
import 'package:intent_digital_hub/IntentDigitalHubService/BALANCA/balanca_command.dart';

class LerPeso extends BalancaCommand {
  final int _qtdLeituras;

  LerPeso(this._qtdLeituras) : super('LerPeso');

  @override
  String functionParameters() {
    return '"qtdLeituras"' ':' '$_qtdLeituras';
  }
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/BALANCA/Commands/LerPeso.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.BALANCA.BalancaCommand

class LerPeso(val qtdLeituras: Int) : BalancaCommand<Int>("LerPeso") {
    override fun functionParameters(): String {
        return "\"qtdLeituras\"" + ":" + this.qtdLeituras
    }
}